### ISSUE TITLE
Add tabcomplete

### ISSUE THREAD

**Issue by deeprobin** [Multiverse/Multiverse-Core#1955]
It would really nice, if the multiverse commands supporting are tabcomplete.

**Comment by Jadiction** [comment]
As long as tab complete is enabled in the YML, then all commands will use tab-complete. Or do you mean when typing MV commands, it should suggest the subcommands?

**Comment by DeepRobin** [comment]
@Jadiction Yes, it should suggest the subcommands. (class implements TabCompletor)

**Comment by Ryckie** [comment]
Hello, how are we with this? For me /mv <tab> doesn't work either. Has it been implemented jet?

**Comment by zax71** [comment]
See the MV5 branch. Most commands are now implemented and we now need to start work on making the other plugins comparable.
